src/serverdlg.cpp

```diff
@@ -331,7 +331,10 @@ lvwClients->setMinimumHeight ( 140 );
     tedWelcomeMessage->setText ( pServer->GetWelcomeMessage() );
 
     // prepare update check info label (invisible by default)
-    lblUpdateCheck->setText ( "<font color=\"red\"><b>" + QString ( APP_NAME ) + " " + tr ( "software upgrade available" ) + "</b></font>" );
+    lblUpdateCheck->setOpenExternalLinks ( true ); // enables opening a web browser if one clicks on a html link
+    lblUpdateCheck->setText (
+        "<font color=\"#c94a55\"><b>" + tr ( "A %1 upgrade is available:&nbsp;" ).arg ( APP_NAME ) +
```

Comment:
Why is the font colour changing?

Reply:
The "red" didn’t fit to the mixer background contrast wise. But we can also agree on another colour.

Reply:
Ideally we'd have a stylesheet with a class...  But if it's got to be an explicit hex code to look right, fine.

Reply:
Yeah. But that's out of scope here probably. I think you should open an issue (and maybe even tag it as "[good first issue](https://github.blog/2020-01-22-how-we-built-good-first-issues/)" since I think it's easy to implement)